static/js/settings_org.js

```diff
@@ -533,7 +533,9 @@ exports.init_dropdown_widgets = () => {
             };
             return item;
         }),
-        subsection: 'notifications',
+        on_update: () => {
+            exports.save_discard_widget_status_handler($(`#org-notifications`));
+        },
```

Comment:
I think it might be cleaner to just rename `subsection` to `settings_subsection` and have that by used to generate a default `on_update` method if one wasn't passed in, if that makes sense?

Reply:
I mainly wanted to keep this as separate as possible. Also, if we add this for topic moving UI as well as the webhook url UI, the majority of usecases wouldn't use the subsection concept.